02-express-tutorial/11-methods.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express")
+const app = express()
+let { people } = require("./data")
+
+//static assets
+app.use(express.static("./methods-public"))
+//parse data
+app.use(express.urlencoded({ extended: false }))
+//parse json
+app.use(express.json())
+//GET
+app.get("/api/people", (req, res) => {
+  res.status(200).json({ success: true, data: people })
+})
+//POST
+app.post("/api/people", (req, res) => {
+  const { name } = req.body
+  if (!name) {
+    return res
+      .status(201)
```

Comment:
I think you want the status to be `400` here to indicate this is an error response.